.github/workflows/e2e.yml

```diff
@@ -26,6 +26,10 @@ on:
         description: 'Name of an existing vlogger image. Leave blank to build oe with the default name'
         type: string
         required: false
+      v2_vertica_image:
+        description: 'Name of a v2 Vertica server image. If blank we will build one using the default name'
+        type: string
+        required: false
```

Comment:
I suggest we move this up ahead of the vlogger image. That way we keep all of the parms related to the vertica-k8s image variants together.